utoipa-gen/src/lib.rs

```diff
@@ -180,6 +180,9 @@ use self::{
 ///   [`HashMap`](std::collections::HashMap) and [`BTreeMap`](std::collections::BTreeMap).
 ///   Free form type enables use of arbitrary types within map values.
 ///   Supports formats _`additional_properties`_ and _`additional_properties = true`_.
+/// * `deprecated` Can be used to mark field as deprecated in the generated OpenAPI spec but
+///   not in the code. If you'd like to mark field as deprecated in the code as well use
+///   Rust's own `#[deprecated]` attribute instead.
```

Comment:
This is good addition. Though this doc should be repeated to the all the types the `deprected` is supported. e.g. for the unnamed structs and enums as well. Also it's worth to check whether `IntoParams` docs are in up-to-date as well.

You can generate the doc with `./scripts/doc.sh` and then just open the generated files with your favorite browser. Files can be found from `target` folder.

Reply:
Furthermore these changes contain the `deprecated` support for `ToSchema` derived types. But query parameters that are derived from `IntoParams` are not affected by this change. They use the same features though and the code can be found from `into_params.rs`. If you find this useful for them as well it might be worthwhile to make some changes there as well. 

All the docs are still in the same place `lib.rs` but just above the `IntoParams` derive trait function.

Reply:
Makes sens. Updated the docs where it was missing, I think.. Let me know